integrations/dd-sdk-android-okhttp-otel/README.md

```diff
@@ -0,0 +1,7 @@
+# Datadog SDK OpenTelemetry extensions for OkHttp
+
+See the dedicated [Datadog SDK instrumentation for OkHttp documentation][1] to learn how track network requests made by `OkHttp` library automatically.
+See the dedicated [Datadog SDK Android for OpenTelemetry documentation][2] to learn how to add a parent span to network requests made by `OkHttp` library.
+
+[1]: https://docs.datadoghq.com/real_user_monitoring/android/advanced_configuration/?tab=kotlin#automatically-track-network-requests
+[2]: https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/android?tab=kotlin
```

Comment:
This link doesn't seem to be working. Is there another page it should open, or is the linked page expected to be merged soon?

Reply:
it will be available soon, the PR is in review on the documentation repo. We will merge that PR once we release this feature. This is the PR: https://github.com/DataDog/documentation/pull/23389